docs/mkdocs/docs/error_messages.md

```diff
@@ -63,7 +63,7 @@ These errors relate to data being pickled, which limits the operations available
 
 Furthermore, it is not possible to partially read/update/append the data using the ArcticDB API or use the QueryBuilder with pickled symbols. 
 
-All of these errors are of type `ArcticNativeCxxException`.
+All of these errors are of type `arcticdb_ext.exceptions.InternalException`.
```

Comment:
It's very sad that we're making such a big deal out of `InternalException` in the public documentation.

I would not mention the exception name at all. Users should not try to catch using the `InternalException` type because when we assign proper error codes to these, the exception type will change.